couchdb/couchdb.go

```diff
@@ -154,12 +166,57 @@ func DeleteDB(dbprefix, doctype string) error {
 }
 
 // ResetDB destroy and recreate the database for a doctype
-func ResetDB(dbprefix, doctype string) error {
-	err := DeleteDB(dbprefix, doctype)
-	if err != nil {
-		return err
+func ResetDB(dbprefix, doctype string) (err error) {
+	err = DeleteDB(dbprefix, doctype)
+	if err == nil {
+		return CreateDB(dbprefix, doctype)
+	}
+	return
```

Comment:
Yep, I just discovered this cool named return and may have been carried away :smile: 
Fixed in 9e1089c6643ac2cfee0405f0ea310abe444e1595